ExampleMod/Content/NPCs/PartyZombie.cs

```diff
@@ -68,6 +68,19 @@ public class PartyZombie : ModNPC
 			return SpawnCondition.OverworldNightMonster.Chance * 0.2f; // Spawn with 1/5th the chance of a regular zombie.
 		}
 
+		public override void AI() {
+			if (NPC.wet) {
+				if (NPC.honeyWet) { // Removes the effects of honey's fall rate making the NPC fall normally in honey
+					NPC.Gravity /= NPC.GravityWetMultipliers[1];
+					NPC.MaxFallSpeed /= NPC.MaxFallSpeedWetMultipliers[1];
+				}
+				else { // Removes water falls speed effects, then adds honey falls speed effects, making the NPC fall at the honey rate in water
+					NPC.Gravity *= NPC.GravityWetMultipliers[1] / NPC.GravityWetMultipliers[0];
```

Comment:
These should use `LiquidID` references. It also seems like you have honey as 1 when it should be 2 (might want to re-order your `GravityWetMultipliers` to match `LiquidID` order